docs/release/new-release-series.md

```diff
@@ -1,28 +1,32 @@
 How to Prepare a New Release Series
 ===================================
 
-!!! warning
-    This document was specifically written for OSG 3.4 and will need to be
-    adapted to work for 3.5, 3.6, etc.
-
+!!!info
```

Comment:
Can we just drop this block? I don't think where this doc comes from is really all that important... and let's not link to my notes in public docs :-)

Reply:
Sure.  I mean not that i really care either way, but yeah i've dropped it.